### DISCUSSION TITLE
Can dokploy manage remote servers

### DISCUSSION THREAD

**Discussion by compgeniuses** [Dokploy/dokploy#211] [General]
From what i have seen, i have noted most of the examples given, asusme dokploy is installing and running apps on the same server.

can it be used to manage and deploy apps to remote server

**Comment by Siumauricio** [comment]
Hi @compgeniuses,

Dokploy cannot manage remote servers for now there is a feature request #22 which is probably you are referring, for now you can use Multi node support which is a similar way to deploy applications to other servers.

**Reply by compgeniuses** [reply]
multi server dashboard will not work, as my idea is to just package dokploy as a dashboard manager into nethserver 8. meaning i still get the same advantages of nethserver with the power of dokploy

**Reply by Siumauricio** [reply]
I can't give you an exact answer to this, but I would assume so from what little I read about nethserver8.
